processor_function/shared/cnab_builder.py

```diff
@@ -113,9 +116,31 @@ def deploy_aci(self):
 
         credential = AzureIdentityCredentialAdapter()
         aci_client = ContainerInstanceManagementClient(credential, self._subscription_id)
+
+        service_bus = ServiceBus()
+        service_bus.send_status_update_message(self._id, "ACI Deployment started", "Deploying ACI container: " + self._container_group_name)
+
         result = aci_client.container_groups.create_or_update(self._resource_group_name, self._container_group_name,
                                                               group)
 
         while result.done() is False:
             logging.info('-- Deploying -- ' + self._container_group_name + " to " + self._resource_group_name)
             time.sleep(1)
+
+        service_bus.send_status_update_message(self._id, "ACI container deployed", "")
```

Comment:
no message?

Reply:
At this point there is no message as the CNAB Azure ACI has just kicked off at this point.